src/simsopt/field/normal_field.py

```diff
@@ -37,7 +43,7 @@ class NormalField(Optimizable):
     """
 
     def __init__(self, nfp=1, stellsym=True, mpol=1, ntor=0,
-                 vns=None, vnc=None):
+                 vns=None, vnc=None, computational_boundary=None):
```

Comment:
The name `computational_boundary` sounds unwieldy. Why not `surface`? There could be use cases for NormalField unrelated to spec for which this surface is not a boundary of anything.